src/components/sbb-tab-group/sbb-tab-group.scss

```diff
@@ -0,0 +1,43 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+.tab-group {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--sbb-spacing-fixed-3x);
+}
+
+.tab-content {
+  @include text;
+
+  color: var(--sbb-color-iron-default);
```

Comment:
I think we should not define any color for the content, because content can be so much different. What do you think @mcilurzo ?

Reply:
Manuel agreed not to define any font style or color for the content and I removed it.